extensions/wikia/WAM/controllers/api/WAMApiController.class.php

```diff
@@ -136,10 +136,10 @@ public function getWAMLanguages() {
 		$wamDay = $this->request->getVal( 'wam_day', null );
 		$wamDates = $this->getMinMaxWamIndexDateInternal();
 
-		if ( empty( $wamDay ) ) {
+		if ( empty( $wamDay ) || $wamDay > $wamDates[ 'max_date' ] ) {
```

Comment:
I wonder if instead of doing `-2 days` in https://github.com/Wikia/app/blob/91752aef233870db296fb66a58eeffe671dd507d/extensions/wikia/WAMPage/WAMPageController.class.php#L99 we can use the `max_date`?

This way it would be more obvious that if no date is passed we want to fallback to latest available one